index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Personal portfolio</title>
+
+  <link rel="Stylesheet" href="./styles.css" />
+</head>
+
+
+<body>
+  <header class="main-header">
+    <a href="#" class="logo">Welcome</a>
+    <nav class="nav">
+      <ul class="nav-menu">
+        <li class="nav-link"><a href="#Portfolio">Portoflio</a></li>
+        <li class="nav-link active"><a href="#About">About</a></li>
+        <li class="nav-link"><a href="#form">Contact</a></li>
+      </ul>
+
+    </nav>
+  </header>
+
+  <main class="container">
+    <section class="main-section">
+      <div class="main-section-title">
+        <h1>Hello, I'm Gilbert Mutai</h1>
+        <h2>
+          I'm a software developer
+        </h2>
```

Comment:
- [ ] Kindly use the same values for font size and font weight for this section. You can inspect a Figma element by clicking on it and the values should appear on your right sidebar. 👍